zuul.d/tempest_multinode.yaml

```diff
@@ -4,7 +4,7 @@
     parent: cifmw-podified-multinode-edpm-base-crc
     timeout: 5400
     abstract: true
-    nodeset: multinode-centos-9-stream-crc
```

Comment:
can we also remove that definition then? https://github.com/openstack-k8s-operators/ci-framework/blob/6ec6e4107436a05283c9ac3b7c286fca6be6ce91/zuul.d/nodeset.yaml#L45

I cannot see it being used somewhere other than here i think? https://github.com/search?q=org%3Aopenstack-k8s-operators%20multinode-centos-9-stream-crc&type=code

Reply:
Thank you Marios, after completing the work to 4.14 I will create a new PR to clean up the unused nodesets.